web_app/forms/sysadmin.py

```diff
@@ -0,0 +1,30 @@
+from django import forms
+from django.contrib.auth.forms import UserChangeForm
+from django.contrib.auth.models import User
+from django.forms.models import ModelForm
+
+
+class AdminUserCreationForm(ModelForm):
+    first_name = forms.CharField(
+        max_length=150,
+    )
+    last_name = forms.CharField(
+        max_length=150,
+    )
+    email = forms.CharField(
+        max_length=254,
+    )
```

Comment:
Does something else need to happen to ensure this follows an email address format?

Reply:
Good point here. I will double check this to make sure it has email address format validation.